omod/src/main/java/org/openmrs/module/registrationapp/page/controller/RegisterPatientPageController.java

```diff
@@ -56,6 +61,17 @@ public void addModelAttributes(PageModel model, Patient patient, AppDescriptor a
         model.addAttribute("enableOverrideOfAddressPortlet",
                 Context.getAdministrationService().getGlobalProperty("addresshierarchy.enableOverrideOfAddressPortlet", "false"));
         model.addAttribute("breadcrumbOverride", breadcrumbOverride);
+
+        // Relationship Types to exclude (Doctor/Patient and Supervisor/Supervisee)
```

Comment:
It was excluded by default in earlier OpenMRS code, so though to exclude here as well. 

Moreover relationships like Doctor <-> Patient and Supervisor <->  Supervisee are irrelevant on patient registration page.